### ISSUE TITLE
[ChannelTabs] Request: Lock a tab so you can't navigate on it

### ISSUE THREAD

**Issue by StevenLeonK** [l0c4lh057/BetterDiscordStuff#132]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

- Lock a tab so you can't navigate on it

**Describe alternatives you've considered**

**Additional context**

**Comment by KiwiTheMemer** [comment]
agreed

**Comment by KiwiTheMemer** [comment]
bump

**Comment by TheChilliPL** [comment]
Yup. Either so that you can't navigate or that you open new tabs on navigating and navigating on them instead
